internal/constants/constants.go

```diff
@@ -153,6 +153,10 @@ MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEf8F1hpmwE+YCFXzjGtaQcrL6XZVT
 JmEe5iSLvG1SyQSAew7WdMKF6o9t8e2TFuCkzlOhhlws2OHWbiFZnFWCFw==
 -----END PUBLIC KEY-----
 `
+
+	// ImageVersionRepositoryURL is the base URL of the repository containing
+	// image version information.
+	ImageVersionRepositoryURL = "https://cdn.confidential.cloud"
```

Comment:
My thinking is that the current implementation makes it very easy to swap out the base if we ever want to support user managed repositories.

Reply:
I agree, but changing it to include /constellation or even /constellation/v1 would allow for even more freedom and less hard-coded path structure a 3rd party would need to adapt.

Not super important though.

Reply:
If we do that, we also have to change the API spec: https://github.com/edgelesssys/constellation/blob/main/rfc/image-discoverability.md#image-api
I would like to leave it for now, unless you see a good reason why a change in the API is beneficial.